e2e/healthcheck_test.go

```diff
@@ -4,23 +4,28 @@ import (
 	"net/http"
 	"os"
 	"os/exec"
-	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
 	"github.com/onsi/gomega/gexec"
 )
 
+const (
+	// kFlag is a keyword to check whether the healthcheck is ready (or process is died)
+	kFlag = "kflag"
+)
+
 var _ = Describe("healthz check should pass", func() {
 	Context("start server with health check", func() {
 		It("should work properly", func() {
 			address, ok := os.LookupEnv("kepler_address")
 			if !ok {
 				address = "localhost:8888"
 				cmd := exec.Command(keplerBin)
-				keplerSession, err = gexec.Start(cmd, GinkgoWriter, GinkgoWriter)
+				keplerSession, err = gexec.Start(cmd, nil, nil)
```

Comment:
Why did you remove the GinkgoWriter?

Reply:
Sorry I just didn't merge with the latest PR.